contracts/multisig/src/multisig_perform.rs

```diff
@@ -177,15 +190,18 @@ pub trait MultisigPerformModule:
                 if let Result::Err(e) = result {
                     sc_panic!(e);
                 }
+
                 OptionalValue::None
-            },
+            }
             Action::SendAsyncCall(call_data) => {
-                let gas_left = self.blockchain().get_gas_left();
+                let gas = call_data
```

Comment:
If I think about it, I am a bit worried on prioritising the call_data gas limit. The thing is, that if for some reason (increased storage on the contract we are calling) the saved gas limit is no longer enough, this action cannot be executed anymore. 
The only solution I see is keeping the gas_limit only as an extra info. That way the front-end can simulate, or increase it if it fails the first time.

Reply:
Well, if that happens, they can simply propose another time. It should be a pretty rare case, I wouldn't complicate the logic of the contract with additional arguments just for this edge case.

Reply:
ok - let's leave it like this and see if we hit this issue along the way

ps - I was not proposing to add another argument. I was proposing to remove this check and only use get_gas_left.
The frontend will use the saved gas_limit as a reference